autoload/fzf/vim.vim

```diff
@@ -633,6 +649,7 @@ function! fzf#vim#gitfiles(args, ...)
     return s:warn('Not in git repo')
   endif
   let prefix = 'git -C ' . fzf#shellescape(root) . ' '
+  let diff_prefix = s:is_linux_like_bash ? 'git -C ' . substitute(root, '^\([A-Z]\):', '/mnt/\L\1', '') . ' ' : prefix
```

Comment:
Shouldn't we still `shellescape` (or `fzf#shellescape`) the `/mnt/...` path in case it contains whitespaces?

Reply:
Hmm, `shellescape` function of Vim is probably not the right choice because this is a Windows vim.

Reply:
> Shouldn't we still `shellescape` (or `fzf#shellescape`) the `/mnt/...` path in case it contains whitespaces?

Understand your point, I'll find a way to deal with it.

Reply:
I did some test with create some whitespaces git repo.
`escape(root, ' ')` or `substitute(root, '\s', '\\ ', 'g')` both worked.

root path with whitespace could be solved, but if there's some whitespaces in filename, I haven't found a solution in `fzf.vim` yet, because I remember this seems need to be solved from the `fzf` side, which I met it before when I was using it in Arch, I don't think we should use whitespace in the filename so I didn't thought it's a issus : )

Reply:
> if there's some whitespaces in filename, I haven't found a solution in `fzf.vim` yet

I'm not sure what you mean by that. `{}` evaluates to an escaped string, so this works as expected.

```sh
echo 'hello' > 'foo bar'
echo 'foo bar' | fzf --preview 'cat {}'
```

> I don't think we should use whitespace in the filename

I agree, but file paths with whitespaces do exist (`Program Files` for example) and we can't just ignore that. Anyway, I believe `escape(root, ' ')` should be good enough for 99% of the cases.

Let me take this over and finish it up hopefully with some improvements.

Reply:
Yeah I did know there's some paths like `Program Files`, so I said `escape(root, ' ')` worked.
What I mead for the whitespaces in the filename is this, for example on Arch:

![Snipaste_2023-01-14_11-58-23](https://user-images.githubusercontent.com/45929738/212450228-c342c4b1-f261-4133-88c9-a9fc5ca2960e.png)

![Snipaste_2023-01-14_11-58-29](https://user-images.githubusercontent.com/45929738/212450232-dcb9c7d1-705b-4a14-ab11-c4c0950df58f.png)

If there's whitespaces in the filename, the preview command occur something, but I think we shouldn't use those filename, so I don't think that's an issus.

And do I need to modify this line to 
`let diff_prefix = s:is_linux_like_bash ? 'git -C ' . escape(substitute(root, '^\([A-Z]\):', '/mnt/\L\1', ''), ' ') . ' ' : prefix`
then rebase to one commit and force push?
And I want to add this to the doc for the `fzf_preview_win_bash`:
`We do not recommend including special characters such as whitespaces in the path if it's not the windows system's native path`

Reply:
#1200 should have fixed it.

Reply:
Ah, that doesn't address the issue in `:GF?` variant.

Reply:
Yeah the `:GitFiles?` needs this too, I'll see what I could do or maybe I could just look forward to your update : )
Then still the question I mentioned above, do I need force push to one commit for updating `diff_prefix` line and doc, or u take all over~

Reply:
> Yeah the `:GitFiles?` needs this too

We'll address this in a separate issue/PR.